ui/store/actions.js

```diff
@@ -1391,6 +1391,19 @@ export function removeCollectible(address, tokenID, dontShowLoadingIndicator) {
   };
 }
 
+export function checkAndUpdateCollectiblesOwnershipStatus() {
+  return async (dispatch) => {
+    try {
+      await promisifiedBackground.checkAndUpdateCollectiblesOwnershipStatus();
+    } catch (error) {
+      log.error(error);
```

Comment:
When is this expected to fail? How does this affect the user? The current approach of logging the error and moving on might make sense of we expect this to fail sometimes but in a way that isn't important to tell the user about. But if we expect this to never fail, we should let the error bubble up to Sentry instead. And if it is important to tell the user, we should consider that in the UI design and expose it somewhere for the user.

The `displayWarning` style of showing the user an error is _awful_ and should be considered as deprecated. It places the error in one of many places around the wallet, where it's shown totally devoid of context, and not translated either.

Reply:
we shouldn't surface any failures to the user. It could potentially fail in an unexpected way that we should bubble to sentry: https://github.com/MetaMask/controllers/blob/main/src/assets/CollectiblesController.ts#L1062. Will remove the `displayWarning` but retain the `log.error(error)`

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13110/commits/f18e1eea775cc11d5395071ec8294f2206b93fac

Reply:
Note that this still won't make it to Sentry. Console logs get included as breadcrumbs (i.e. as context for other errors), but they aren't recorded as errors themselves. You'd need to remove the `try { .. } catch` block altogether to let it bubble up to Sentry. Unhandled exceptions/rejections are captured.